extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -89,6 +91,8 @@ export default class MainController implements Disposable {
 				for (const file of files) {
 					await this.projectsController.openProject(file);
 				}
+
+				this.projectsController.focusProject(this.projectsController.projects[0]);
```

Comment:
always [0]? what happens for the second project?

Reply:
Good catch; meant the first of the newly-opened projects